hypothesis-python/docs/reference/strategies.rst

```diff
@@ -1,15 +1,216 @@
+.. _strategies:
+
+Strategies reference
+====================
+
+Primitives
+----------
+
+.. autofunction:: hypothesis.strategies.none
+.. autofunction:: hypothesis.strategies.nothing
+.. autofunction:: hypothesis.strategies.just
+.. autofunction:: hypothesis.strategies.booleans
+
+Numeric
+-------
+
+.. autofunction:: hypothesis.strategies.integers
+.. autofunction:: hypothesis.strategies.floats
+.. autofunction:: hypothesis.strategies.complex_numbers
+.. autofunction:: hypothesis.strategies.decimals
+.. autofunction:: hypothesis.strategies.fractions
+
+Text
+----
+
+.. autofunction:: hypothesis.strategies.text
+.. autofunction:: hypothesis.strategies.characters
+.. autofunction:: hypothesis.strategies.from_regex
+.. autofunction:: hypothesis.strategies.binary
+.. autofunction:: hypothesis.strategies.emails
+.. autofunction:: hypothesis.strategies.uuids
+.. autofunction:: hypothesis.strategies.ip_addresses
```

Comment:
These aren't strings, they're [`IPAddress` objects](https://docs.python.org/3/library/ipaddress.html), and thus probably belong in misc.

Reply:
same with UUIDS then?

Reply:
🤦‍♂️ yes.  We could add a _see also_ note at the top of the section though, noting that you can access uuid / ip address strings via `st.uuids(...).map(str)` etc.